src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -215,6 +215,11 @@ public abstract class Opcode
             "+-------------------------------------------------------------------------+\n";
 
         public int Id { get { return id; } }
+        /// <summary>
+        /// How far to jump to the next opcode.  1 is typical (just go to the next opcode),
+        /// but in the case of jump and branch statements, it can be other numbers.  This will
+        /// be ignored if the CPU has been put into a yield state with CPU.YieldProgram().
+        /// </summary>
         public int DeltaInstructionPointer { get; protected set; }
```

Comment:
Added the above comment to clarify the new technique will NOT be to have OpcodeWait set its DeltaInstructionPointer to 0, but rather to have the CPU itself go ahead and move the IP to the next instruction, but then not execute it yet until the wait period is over.  I didn't like the way it used to spin on the OpcodeWait, re-executing it each update, instead of just not executing anything until the time is over.